src/rules/at-rule-blockless-newline-after/README.md

```diff
@@ -0,0 +1,53 @@
+# at-rule-blockless-newline-after
+
+Require a newline after blockless @rules.
+
+```css
+a {}
```

Comment:
Can you change this example to

```css
@import url("x.css");
@import url("y.css");
/**                 ↑
 * The newline after these semicolons */
```

Also note the single space before "The newline..."